hack/update-openapi-spec.sh

```diff
@@ -96,6 +97,12 @@ if ! kube::util::wait_for_url "https://${API_HOST}:${API_PORT}/healthz" "apiserv
   exit 1
 fi
 
+kube::log::status "Updating aggregated discovery"
+
+rm -fr "${DISCOVERY_ROOT_DIR}"
+mkdir -p "${DISCOVERY_ROOT_DIR}"
+curl -kfsS -H 'Authorization: Bearer dummy_token' -H 'Accept: application/json;g=apidiscovery.k8s.io;v=v2beta1;as=APIGroupDiscoveryList' "https://${API_HOST}:${API_PORT}/apis" | jq -S . > "${DISCOVERY_ROOT_DIR}/aggregated_v1beta1.json"
```

Comment:
why this is v2beta1 and the file v1beta1? is there are typo or are different things?

Reply:
oops, typo

Reply:
fixed